corehq/apps/export/models/new.py

```diff
@@ -559,10 +559,6 @@ def generate_instance_from_schema(cls, schema, saved_export=None):
         )
         group_schemas = schema.group_schemas
 
-        inferred_schema = instance._get_inferred_schema()
-        if inferred_schema:
-            group_schemas.extend(inferred_schema.group_schemas)
-
```

Comment:
this was totally wrong, i needed to do this in the schema generation, not the instance generation so we can merge the schemas